cluster-up/cluster/k8s-1.19/provider.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -e
+source ${KUBEVIRTCI_PATH}/cluster/k8s-provider-common.sh
```

Comment:
please add extra line

Reply:
This is just the result of the bump from vendoring kubevirtci. In case you are interested, please have a look here kubevirt/kubevirtci#434